dojo/templates/dojo/product_type.html

```diff
@@ -42,44 +42,65 @@ <h3 class="has-filters">
                            class="tablesorter-bootstrap table table-condensed table-striped">
                         <thead>
                         <tr>
+                            <th></th>
                             <th>{% dojo_sort request 'Product Type' 'name' 'asc' %}</th>
-                            <th> Product count</th>
-                            <th> Active (Verified) findings</th>
-                            {% if user.is_staff %}
-                                <th> Actions</th>{% endif %}
+                            <th>Product count</th>
+                            <th>Active (Verified) findings</th>
+                            <th>Critical product</th>
+                            <th>Key product</th>
                         </tr>
                         </thead>
                         <tbody>
                         {% for pt in pts %}
                             <tr>
-                                <td><a href="{% url 'product' %}?prod_type={{ pt.id }}">{{ pt.name }}</a></td>
-                                <td><a href="{% url 'product_type_metrics' pt.id %}"> {{ pt.prod_count }} </a></td>
+                                <td>
+                                    <ul>
+                                        <li class="dropdown" style="list-style:none;position:absolute">
+                                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="true"  id="dropdownMenuPT">&nbsp;<b class="fa fa-ellipsis-v"></b>&nbsp;</a>
```

Comment:
"dropdownMenuPT" => snake_case ?

Reply:
We find both snake_case and CamelCase as id. I changed this id to a proper CamelCase now.

Reply:
I agree all ids are not consistent. I will share with the team